kalite/updates/views.py

```diff
@@ -49,7 +49,7 @@ def update_context(request):
 @render_to("updates/update_videos.html")
 def update_videos(request, max_to_show=4):
     context = update_context(request)
-    messages.warning(request, 'For low-powered devices like the Raspberry Pi, please download videos one at a time.')
+    messages.warning(request, _('For low-powered devices like the Raspberry Pi, please download videos one at a time.'))
```

Comment:
As I was scanning through strings I noticed that this wasn't i18nized yet. Here you go!